bitwarden_license/src/Commercial.Infrastructure.EntityFramework/Repositories/ProjectRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using AutoMapper;
+using Bit.Core.Repositories;
+using Bit.Infrastructure.EntityFramework.Models;
+using Bit.Infrastructure.EntityFramework.Repositories;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Bit.Commercial.Infrastructure.EntityFramework.Repositories
+{
+    public class ProjectRepository : Repository<Core.Entities.Project, Project, Guid>, IProjectRepository
+    {
+        public ProjectRepository(IServiceScopeFactory serviceScopeFactory, IMapper mapper)
+            : base(serviceScopeFactory, mapper, db => db.Project)
+        {
+
+        }
+
+        public override async Task<Core.Entities.Project> GetByIdAsync(Guid id)
+        {
+            using (var scope = ServiceScopeFactory.CreateScope())
+            {
+                var dbContext = GetDatabaseContext(scope);
+                var project = await dbContext.Project
+                                        .Where(c => c.Id == id && c.DeletedDate == null)
+                                        .FirstOrDefaultAsync();
+                return Mapper.Map<Core.Entities.Project>(project);
+            }
+        }
+
+        public async Task<IEnumerable<Core.Entities.Project>> GetManyByOrganizationIdAsync(Guid organizationId)
+        {
+            using (var scope = ServiceScopeFactory.CreateScope())
+            {
+                var dbContext = GetDatabaseContext(scope);
+                var project = await dbContext.Project
+                                        .Where(c => c.OrganizationId == organizationId && c.DeletedDate == null)
+                                        .ToListAsync();
```

Comment:
Do we want to return projects by a specific order? 
For example, order by revision date.

Reply:
Yes, good idea. In the ticket Oscar confirmed we should order by revision date.